cmd/opm/bundle/generate.go

```diff
@@ -0,0 +1,60 @@
+package bundle
+
+import (
+	"github.com/operator-framework/operator-registry/pkg/lib/bundle"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+type AnnotationMetadata struct {
```

Comment:
@gallettilance

Reply:
@dinhxuanvu Instead of putting this type in the cmd package, can we expose it in one of the packages in `pkg/` so that it can be imported in multiple places? The database function that parses these bundles will need to use this type to read the annotations in the other direction.

Reply:
@kevinrizza The type has been moved to bundle package already but I forgot to remove it here.